app/assets/javascripts/admin/ck_editor_initiate.js

```diff
@@ -0,0 +1,5 @@
+ClassicEditor
```

Comment:
let's use prototype JS. Also ensure that this code only gets executed after DOM load